code/go/0chain.net/chaincore/httpclientutil/http_client_util_test.go

```diff
@@ -0,0 +1,1050 @@
+package httpclientutil
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+
+	"0chain.net/chaincore/block"
+	"0chain.net/chaincore/node"
+	"0chain.net/chaincore/state"
+	"0chain.net/core/common"
+	"0chain.net/core/encryption"
+	"0chain.net/core/logging"
+	"0chain.net/core/util"
+	"0chain.net/mocks"
+)
+
+func init() {
+	block.SetupEntity(&mocks.Store{})
+	logging.InitLogging("development")
```

Comment:
httpClient is a global varable which is defined once in [http_client_util.go.init](https://github.com/0chain/0chain/blob/master/code/go/0chain.net/chaincore/httpclientutil/http_client_util.go#L65). We should do the same thing here. It should be ok to remove the `t.Skip()`s after this.